internal/api/auth.go

```diff
@@ -87,6 +94,11 @@ func Signup(s storage.Store) http.HandlerFunc {
 
 		go app.SendMail([]string{viper.GetString("email.admin")}, subject, body)
 
+		confirmationBody := "Last step for use Passwall\n\n"
+		confirmationBody += "Confirmation link: " + "http://" + viper.GetString("server.domain") + ":" + viper.GetString("server.port")
+		confirmationBody += "/auth/confirm/" + userDTO.Email + "/" + confirmationCode
+
+		go app.SendMail([]string{userDTO.Email}, "Passwall Email Confirmation", confirmationBody)
```

Comment:
I think the messages should be read from the localization files

Reply:
I don't see any localization implement on this repository. I think this is another issue for repository.

Reply:
There are localization files under storage folder but I'm not fully looked the code base how to use it

Reply:
Localization is still in progress. For now we can approve inline comments.